src/python/__template__.py

```diff
@@ -62,22 +62,23 @@
 """
 
 
-def function_with_types_in_docstring(param1, param2):
-    """Example function with types documented in the docstring.
+def function_with_types_in_docstring(param1: int, param2: str) -> bool:
```

Comment:
then write 2 def:
retain function_with_types_in_docstring
and add this to `function_with_types`